xcube/core/store/store.py

```diff
@@ -690,3 +689,19 @@ def deregister_data(self, data_id: str):
         Raises:
             DataStoreError: If an error occurs.
         """
+
+
+class PreloadedDataStore(DataStore):
+    """A preload data store is a multable data store which contains the preload handle.
+
+    Instances of this class are returned by the ``DataStore.preload_data()`` method.
+    """
+
+    @property
+    @abstractmethod
+    def preload_handle(self) -> PreloadHandle:
+        """The preload handle that can be used to observe the preload progress.
+        Implementors of this interface may use a `ExecutorPreloadHandle` or consider
+        returning a `NullPreloadHandle` if the progress is not observable.
+        """
+        pass
```

Comment:
Remove line. `pass` is redundant because we have a docstring which acts as no-op statement.